courts_db/data/courts.json

```diff
@@ -24330,7 +24519,8 @@
             "State of New Jersey State Board of Taxes And Assessment",
             "New Jersey Division of Tax Appeals",
             "New Jersey State Board of Taxes and Assessment",
-            "New Jersey Department of Taxation and Finance"
+            "New Jersey Department of Taxation and Finance",
```

Comment:
this is different than the tax board I think.. can we see if all these are different parts of the process